src/ImageRunModal.jsx

```diff
@@ -824,222 +824,224 @@ export class ImageRunModal extends React.Component {
+                            }
+
+                            <FormGroup fieldId='run-image-dialog-command' label={_("Command")}>
+                                <TextInput id='run-image-dialog-command'
+                                    value={dialogValues.command || ''}
+                                    onChange={(_, value) => this.onValueChanged('command', value)} />
+                            </FormGroup>
+
+                            <FormGroup fieldId="run=image-dialog-tty">
+                                <Checkbox id="run-image-dialog-tty"
+                                    isChecked={this.state.hasTTY}
+                                    label={_("With terminal")}
+                                    onChange={(_event, checked) => this.onValueChanged('hasTTY', checked)} />
+                            </FormGroup>
+
+                            <FormGroup fieldId='run-image-dialog-memory' label={_("Memory limit")}>
+                                <Flex alignItems={{ default: 'alignItemsCenter' }} className="ct-input-group-spacer-sm modal-run-limiter" id="run-image-dialog-memory-limit">
+                                    <Checkbox id="run-image-dialog-memory-limit-checkbox"
+                                        isChecked={this.state.memoryConfigure}
+                                        onChange={(_event, checked) => this.onValueChanged('memoryConfigure', checked)} />
                                     <NumberInput
-                                        id="run-image-cpu-priority"
-                                        value={dialogValues.cpuShares}
-                                        onClick={() => !this.state.cpuSharesConfigure && this.onValueChanged('cpuSharesConfigure', true)}
-                                        min={2}
-                                        max={262144}
-                                        isDisabled={!this.state.cpuSharesConfigure}
-                                        onPlus={() => this.onPlusOne('cpuShares')}
-                                        onMinus={() => this.onMinusOne('cpuShares')}
-                                        minusBtnAriaLabel={_("Decrease CPU shares")}
-                                        plusBtnAriaLabel={_("Increase CPU shares")}
-                                        onChange={ev => this.onNumberValue('cpuShares', ev.target.value, 2)} />
+                                        value={dialogValues.memory}
+                                        id="run-image-dialog-memory"
+                                        min={0}
+                                        isDisabled={!this.state.memoryConfigure}
+                                        onClick={() => !this.state.memoryConfigure && this.onValueChanged('memoryConfigure', true)}
+                                        onPlus={() => this.onPlusOne('memory')}
+                                        onMinus={() => this.onMinusOne('memory')}
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2246-4debadf1-20250829-124849-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)